apps/desktop/electron-builder-ms.config.js

```
@@ -1,64 +1,24 @@
+  ...baseElectronBuilderConfig,
   'nsis': {
     'oneClick': false,
-    'installerSidebar': 'build/static/images/icons/installerSidebar.bmp',
+    'installerSidebar': 'app/build/static/images/icons/installerSidebar.bmp',
     'deleteAppDataOnUninstall': true,
   },
   'win': {
     'extraResources': [
       {
-        'from': 'build/static/bin/bridge/win-${arch}',
+        'from': 'app/build/static/bin/bridge/win-${arch}',
```

ESLint

[error] Unexpected template string expression.

(no-template-curly-in-string)